pyproject.toml

```diff
@@ -187,13 +187,9 @@ filterwarnings = [
     "ignore:Deprecated call to",
     "ignore:pkg_resources is deprecated as an API",          # Remove on https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2181 has been fixed
 ]
-addopts = "-vs --cov-report term-missing --cov-report xml --dist loadscope --junitxml=pytest-junit.xml"
+addopts = "-v --cov-report term-missing --cov-report xml --dist loadscope --junitxml=pytest-junit.xml"
```

Comment:
@dgarros As discussed, this would enable logging only for failed tests, so we can keep `WARNING` level without having too much logs in a pytest CI report. We can still enable logging on successful tests for local development by passing `-s` option.